### ISSUE TITLE
Wikidata reconciliation problem

### ISSUE THREAD

**Issue by loraxian** [OpenRefine/OpenRefine#3232]
I can reconcile columns against Wikidata OK, but when I try to create a schema none of the columns will drag down. The columns are 100% reconciled, and I am still able to reconcile data from within the project OK. I have tried opening previous projects that generated edits successfully, and the reconciliation service fails to connect, and the schema has disappeared from them and is unable to be recreated (screenshot from a previously successful project attached).
<img width="1280" alt="Screen Shot 2020-09-30 at 3 37 50 PM" src="https://user-images.githubusercontent.com/68826982/94637071-22faa880-0333-11eb-91e0-2fbabfc6ec98.png">

I've restarted OpenRefine, and Firefox, tried Chrome, restarted the computer, and reinstalled OpenRefine, to no avail. I am using 3.4.1 on Firefox (but problem persists in Chrome).
Any suggestions appreciated, thanks,
Tamsin

**Comment by wetneb** [comment]
Which reconciliation service have you used to reconcile your columns?

At the moment https://wdreconcile.toolforge.org/en/api returns an HTTP 504 error for me. This reconciliation service is known to be unreliable and is unmaintained. We would like to switch away from it: https://github.com/OpenRefine/OpenRefine/issues/3080

Perhaps one temporary workaround would be to reconfigure Wikidata to use "https://wikidata.reconci.link/en/api" in the schema, which should be possible by writing a new manifest for Wikidata with this reconciliation service. But I am not sure if we let the user do that when there is already one Wikibase configured with the same MediaWiki endpoint.

**Comment by loraxian** [comment]
Thank you. I had tried every reconciliation service I could find! (I remember it changed at the end of July). I had two services marked as old, which would not delete, and one to https://wikidata.reconci.link/en/api that shows as wikidata reconci.link (en). I have since managed to delete the services marked as "old" (with difficulty because the services tab keeps bouncing across the screen). It has not changed the problem, I can still reconcile cells but cannot create a schema.
